src/main/java/net/minecraft/launchwrapper/injector/AlphaVanillaTweakInjector.java

```diff
@@ -16,10 +15,21 @@
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class AlphaVanillaTweakInjector implements IClassTransformer {
+    // The main classloader
+    private static LaunchClassLoader launchClassLoader;
+
     public AlphaVanillaTweakInjector() {
+        List<ITweaker> tweaks = (List<ITweaker>) Launch.blackboard.get("Tweaks");
+        for (ITweaker tweaker : tweaks) {
+            if (tweaker instanceof AlphaVanillaTweaker) {
+                launchClassLoader = ((AlphaVanillaTweaker) tweaker).launchClassLoader;
```

Comment:
Transformers are loaded inside of the `LaunchClassLoader`. Instead of sneakily passing the class loader around, you can just use `getClass().getClassLoader()` and cast the class loader to `LaunchClassLoader`.